### PR TITLE
members div

### PR DESCRIPTION
**Pull request by yuvraajnarula**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -14,19 +14,41 @@ body {
     margin-right: auto;
     float:right;
 }
-
+nav{
+    z-index: 99;
```
Comment (ion05) [inline main.css:18]:
```suggestion
    z-index: 99;
    position:fixed;
```

File: main.css [hunk 2]
```diff
@@ -50,11 +72,181 @@ width: 100%;
 }
 
 .aim-head{
-    font-family: "Antonio", sans-serif;
+    font-family: "Ubuntu", sans-serif;
```
Comment (ion05) [inline main.css:75]:
```suggestion
    font-family: "Antonio", sans-serif;
```

File: main.css [hunk 3]
```diff
@@ -14,19 +14,41 @@ body {
...
+    position: absolute;
+    color: #eee;
+    top:62.5%;
+    text-align: right;
+    left: 52.5%;
 }
 /* Aim Section */
-
+.aim{
+    left: -3%;
```
Comment (ion05) [inline main.css:50]:
```suggestion
```

I will handle this

**Review by ion05 — CHANGES_REQUESTED** [review]

File: main.css (hunk 2 above)
Comment (yuvraajnarula) [reply main.css:75]: NO not happening already told you to keep maximum 3 fonts not more than that

Comment (ion05) [reply main.css:72]: We are using only 3 fonts.. nd this just adds more

File: main.css (hunk 1 above)
Comment (yuvraajnarula) [reply main.css:18]: fixed-top added as a class no need to worry.

File: main.css (hunk 2 above)
Comment (ion05) [reply main.css:72]: Antonio looks good on headings .. ubuntu is just bland

File: index.html [hunk 4]
```diff
@@ -8,20 +8,23 @@
...
         rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Antonio&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <!-- Navbar -->
-    <nav class="navbar navbar-expand-lg navbar-dark bg-dark" id="nav">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark p-4 " id="nav">
```
[the remark this answers is not in the document]
Comment (ion05) [reply index.html]: u didnt remove it

File: main.css (hunk 2 above)
Comment (yuvraajnarula) [reply main.css:72]: image
Antonio^
image
Ubuntu^

Comment (ion05) [reply main.css:72]: Yeah but Antonio looks good with all heading all caps. usse hi rakhle na

Comment (ion05) [reply main.css:72]: the first one aint antonio

Comment (ion05) [reply main.css:72]: u dlted the importing link for it

**Review by ion05 — CHANGES_REQUESTED** [review]
Combine Footer

File: index.html [hunk 5]
```diff
@@ -96,12 +100,283 @@ <h2 class="aim-head "> OUR AIM </h2>
...
+        <div class="container">
+            <div class="row">
+              <center>
+                    <h3>Our Social Links</h3>
+                    <p>Do check our social pages to get updated for upcoming events. For more info these are our links given below.</p>
+                </div>
+         
+                <div class="col item social"><a href="#"><i class="icon fa fa-instagram"></i></a><a href="#"><i class="icon fa fa-envelope"></i></a><a href="#"><i class="icon fa fa-facebook"></i></a><a href="#"><i class="icon fa fa-twitter"></i></a></div>
+            </div>
+            <p class="copyright">Amity International School Sec-46 © 2021</p>
```
Comment (ion05) [inline index.html:374]:
```suggestion
            <p class="copyright">Amity International School Sector-46 © 2021</p>
```

File: index.html [hunk 6]
```diff
@@ -96,12 +100,283 @@ <h2 class="aim-head "> OUR AIM </h2>
...
+    <!--forms-->
+    <div class="form-div">
+        <div class="container p-4">
+            <div class="row">
+
+                    <h1>Contact Us</h1>
 
+                </div>
+            <div class="row">
+                    <h4 style="text-align:left">We'd love to hear from you!</h4>
```
Comment (ion05) [inline index.html:326]: Switch fonts of Contact us and we'd love to hear

**Review by ion05 — CHANGES_REQUESTED** [review]

**Pull request merged**